Hello_Object_Oriented_Due_Dec_1/renderBoard.js

```diff
@@ -0,0 +1,132 @@
+
+
+class Task {
+  constructor(taskName='', assignee='') {
+    this.taskName = taskName;
+    this.assignee = assignee;
+  }
+
+  render() {
+    if (this.assignee != '' && this.taskName != '') {
+
+      return `${this.taskName} ${this.assignee}`;
+    }
+    else if (this.taskName != '') {
+
+      return `${this.taskName}`;
+    } else {
+      return '';
+    }
+  }
+}
+
+
+class List {
+
+  constructor(listName) {
+    this.listName = listName;
+    this.lists = []; // store tasks
+  }
+  addTask (task) {
+    this.lists.push(task);
+    return this;
+  }
+  removeTask(task) {
+    for (var i = 0; i < this.lists.length; i++) {
```

Comment:
It is not considered bad practice to be using `var`. Please use `const` for values that will not change, and `let` for values that do get reassigned (in this case, you would use `let i = 0`)